src/client/pages/PageNotFound/index.js

```diff
@@ -9,7 +9,7 @@ function PageNotFound() {
   return (
     <div id="overflow-wrapper">
       <Header />
-      <main>
+      <main id="certification-page">
```

Comment:
Unclear if PageNotFound is a certification-page or not, but I guess this is OK. I think it's also OK to not add the id here.